### DISCUSSION TITLE
WebUI - How to use

### DISCUSSION THREAD

**Discussion by jivetek** [hnimminh/libresbc#120] [General]
If this is incorrect let me know, but it looks like the idea is to download the /webui directory to your local machine, install go on your machine (https://go.dev/dl/) and then run it with (from the webui directory) "go run main.go -libresbc https://your-sbc-ip-address-here:8443"

I had never used go before so I spent a couple of hours poking around until I figured those parts out. 

Thought the info might be helpful to others

**Comment by Jakihiko** [comment]
thanks @jivetek for the guide, can you post some screenshot there?

**Comment by poldy42** [comment]
Good morning, 
just startet to play around with this nice tool, and found your hint @jivetek - many thanks.

Looks like that the webui is more or less a web based version of a tool like postman. Usages is rather easy:

- as Matthew mentioned you should install go an run the commandline from the downloaded webui-Folder:
![1](https://github.com/hnimminh/libresbc/assets/122225698/58e88a55-7d93-41be-a195-dd91bef0c1d2)

- now you can open a webbrowser and open http://127.0.0.1:8088 (not 0.0.0.0 as mentoned in cmd window !)
![2](https://github.com/hnimminh/libresbc/assets/122225698/19c0ef69-6790-40e5-a17c-08be15711ca4)

Happy configuring :-)

Cheers - Michael

**Comment by pcaveness** [comment]
Be sure to point to your SSL cert, key, and domain.

`go run main.go -tls -certs /etc/ssl/certs/{certificate}.crt:/etc/ssl/private/{key}.key -domain company.com -libresbc https://127.0.0.1:8443`
